java/mtsj/core/pom.xml

```diff
@@ -320,4 +297,57 @@
     </plugins>
   </build>
 
+  <profiles>
+    <profile>
+      <id>postgresql</id>
+      <properties>
+        <mts.database.name>postgresql</mts.database.name>
+        <mts.database.profile>postgresql</mts.database.profile>
+        <mts.database.groupId>org.postgresql</mts.database.groupId>
+        <mts.database.artifactId>postgresql</mts.database.artifactId>
+        <mts.database.version>9.4.1211.jre7</mts.database.version>
+      </properties>
+    </profile>
+    <profile>
+      <id>mariadb</id>
+      <properties>
+        <mts.database.name>mariadb</mts.database.name>
+        <mts.database.profile>mariadb</mts.database.profile>
+        <mts.database.groupId>org.mariadb.jdbc</mts.database.groupId>
+        <mts.database.artifactId>mariadb-java-client</mts.database.artifactId>
+        <mts.database.version>1.5.4</mts.database.version>
+      </properties>
+    </profile>
+    <profile>
+      <id>mysql</id>
+      <properties>
+        <mts.database.name>mysql</mts.database.name>
+        <mts.database.profile>mysql</mts.database.profile>
+        <mts.database.groupId>mysql</mts.database.groupId>
+        <mts.database.artifactId>mysql-connector-java</mts.database.artifactId>
+        <mts.database.version>6.0.6</mts.database.version>
+      </properties>
+    </profile>
+    <profile>
+      <id>oracle</id>
+      <properties>
+        <mts.database.name>oracle</mts.database.name>
+        <mts.database.profile>oracle</mts.database.profile>
+        <mts.database.groupId>com.oracle</mts.database.groupId>
+        <mts.database.artifactId>ojdbc7</mts.database.artifactId>
+        <mts.database.version>12.1.0.2</mts.database.version>
```

Comment:
groupId "com.oracle" is not getting resolved when trying to download from oracle maven repository. Changing it to "com.oracle.jdbc" as per oracle guide it got downloaded.  Secondly, when trying to build the project tests fail, with error related to parsing of _META-INF/cxf/cxf.xml_. I also tried to build the project with different artifactId and version, in this case "ojdbc8" and "12.2.0.1" . And was able to build the project successfully.

Reply:
@AbhayChandel thanks for your feedback and testing. That is great news. For Oracle the JDBC driver is IMHO not available in the public maven central repository:
http://repo1.maven.org/maven2/com/oracle/
Unfortunately therefore there are no standard GAV coordinates for a oracle JDBC driver dependency. What I preconfigured are best-practices used in various projects. However, you need to deploy the driver to your own artifact repo (nexus, artifactory, or whatever).
If it worked for your using `com.oracle.jdbc` I assume that this is only by accident as you did a project before that used this groupId and therefore the driver is in your local repository.
An alternative would IMHO only be to drop Oracle support at all. However, the current solution is at least a 70% solution and would probably need some more documentation.

For the `cxf.xml` I can not see how this may be related to oracle or my changes at all.

Reply:
As I got no response, I searched myself. It seems you were referring to this:
https://blogs.oracle.com/dev2dev/get-oracle-jdbc-drivers-and-ucp-from-oracle-maven-repository-without-ides
as well as this:
https://docs.oracle.com/middleware/1213/core/MAVEN/config_maven_repo.htm#MAVEN9013
Correct?

Reply:
Hi @hohwille, below is the guide for getting Oracle JDBC drivers. Let me know if you have any questions.
https://blogs.oracle.com/dev2dev/get-oracle-jdbc-drivers-from-the-oracle-maven-repo-netbeans-eclipse-intellij